app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! WelcomeScreen View.
+//!
+//! It is opened when the IDE launches without any project or entry point selected. It
+//! displays a list of available projects, template cards and "new project" button.
+
+#![warn(missing_docs)]
+
+
+mod side_menu;
+mod template_cards;
+
+use ensogl::prelude::*;
+
+use crate::side_menu::SideMenu;
+use crate::template_cards::TemplateCards;
+
+use enso_frp as frp;
+use ensogl::application;
+use ensogl::application::Application;
+use ensogl::display;
+use ensogl::display::DomSymbol;
+use ensogl::system::web;
+use ensogl::system::web::NodeInserter;
+use std::rc::Rc;
+use wasm_bindgen::closure::Closure;
+use web_sys::HtmlDivElement;
+use web_sys::MouseEvent;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+const TEMPLATES_VIEW_ROOT: &str = "enso-internal-templates-view";
+const CONTAINER: &str = "enso-internal-container";
+const SIDE_MENU: &str = "enso-internal-side-menu";
+const CONTENT: &str = "enso-internal-content";
+const CARDS: &str = "enso-internal-cards";
+const CARD: &str = "enso-internal-card";
+const ROW: &str = "enso-internal-row";
+const CARD_SPREADSHEETS: &str = "enso-internal-card-spreadsheets";
+const CARD_GEO: &str = "enso-internal-card-geo";
+const CARD_VISUALIZE: &str = "enso-internal-card-visualize";
```

Comment:
```suggestion
pub mod css_classes {
    const TEMPLATES_VIEW_ROOT: &str = "enso-internal-templates-view";
    const CONTAINER: &str = "enso-internal-container";
    const SIDE_MENU: &str = "enso-internal-side-menu";
    const CONTENT: &str = "enso-internal-content";
    const CARDS: &str = "enso-internal-cards";
    const CARD: &str = "enso-internal-card";
    const ROW: &str = "enso-internal-row";
    const CARD_SPREADSHEETS: &str = "enso-internal-card-spreadsheets";
    const CARD_GEO: &str = "enso-internal-card-geo";
    const CARD_VISUALIZE: &str = "enso-internal-card-visualize";
}
```
Initially, I thought that those are element ids.

Reply:
Done, in slightly different way: module is name `css_class`, as `crate::css_class::FOO` feels more natural to me. Also added submodule `css_id` with a single member - the only one that is used in CSS stylesheet. All other ids removed as unnecessary. (JS code heavily relies on them, but we don't)

Reply:
You are right, the module name should be singular.